src/components/Card/Card.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import CardHeader from './CardHeader';
+import CardContent from './CardContent';
+import CardFooter from './CardFooter';
+
+type CardProps = {
+  title: string;
+  description: string;
+  content: { label: string; value: string }[];
+  footer?: boolean;
+};
+
+const Card = ( 
+{
+  title,
+  description,
```

Comment:
Props should be only header content and footer. The component in the card itself should be passed from the outside